pkg/driver/common/operator/hooks.go

```diff
@@ -85,8 +86,9 @@ func withHyperShiftNodeSelector(c *clients.Clients) (dc.DeploymentHookFunc, []fa
 
 		return nil
 	}
+	genInformer, _ := c.ControlPlaneHypeInformer.ForResource(hostedControlPlaneGVR)
```

Comment:
Again, a typed informer would be more appropriate
```suggestion
	hcpInformer := c.ControlPlaneHypeInformer.Hypershift().V1beta1().HostedControlPlanes().Informer()
```